cluster-scope/base/rbac.authorization.k8s.io/rolebindings/ray-operator-rolebinding/rolebinding.yaml

```diff
@@ -0,0 +1,12 @@
+kind: RoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: ray-operator-rolebinding
```

Comment:
@larsks because we didn't specify a namespace for this rolebinding looks like it ended up in some default namespace.

```
naved@Naveds-MacBook-Pro ~ % oc get rolebinding -n open-cluster-management-agent
NAME                                                    ROLE                                                         AGE
open-cluster-management:klusterlet-registration:agent   Role/open-cluster-management:klusterlet-registration:agent   271d
ray-operator-rolebinding                                Role/ray-operator-role                                       7d3h
system:deployers                                        ClusterRole/system:deployer                                  271d
system:image-builders                                   ClusterRole/system:image-builder                             271d
system:image-pullers                                    ClusterRole/system:image-puller                              271d
```

@acemaster I imagine these role-bindings need to go into the `remote-sensing` namespace?